app/helpers/comments_helper.rb

```diff
@@ -0,0 +1,2 @@
+module CommentsHelper
```

Comment:
Safe to delete blank helpers like `comments_helper.rb`. Helps keep codebase tidier.